lib/libc/unistd/lib_usleep.c

```diff
@@ -95,7 +95,8 @@
  *   due to the scheduling of other activity by the system.
  *
  *   The 'usec' argument must be less than 1,000,000. If the value of
- *   'usec' is 0, then the call has no effect.
+ *   'usec' is 0, then the call has no effect. It operates in ticks, so if
```

Comment:
If the value of 'usec' is less than the value of `tick`, CONFIG_USEC_PER_TICK, then this function operates with the value of tick.

Reply:
@btheosam Could you review this PR?

Reply:
All good, minor clarification it is valid only in tick-less mode.